packages/x-components/tests/e2e/cucumber/history-queries.feature

```diff
@@ -81,7 +81,6 @@ Feature: History queries component
     When  search-input is focused
     And   history query number <historyQueryItem> is clicked
     And   clear search button is pressed
-    And   search-input is focused
```

Comment:
I understand this step call was removed from different files because that action is now done automatically. The step is not used for anything else so is no longer needed so remove it from the `common-steps.spec.ts` file.

Reply:
The action is now done automatically but only when the user clicks on the clear search button. So I understand the step is still useful in other cases.

Reply:
I had to ask @mavmaf too xD. The  `search-input is focused` is quite confuse too me, it looks like a check rather than an action.

Reply:
My bad, ran the search for the step with a file mask, the step is used in other places 🤐